### PR TITLE
Improve INSTALL.md

### PR DESCRIPTION
**Pull request by techboyg5**
I also fixed the capitalization of "JavaScript", and clarified that PHP 7.0 or higher do not need a source of randomness.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -11,20 +11,16 @@ options](#configuration) to adjust as you see fit.
...
+- PHP version 7.0 or above
+  - Or PHP version 5.6 AND _one_ of the following sources of cryptographically safe randomness:
+    - [Libsodium](https://download.libsodium.org/libsodium/content/installation/) and it's [PHP extension](https://paragonie.com/book/pecl-libsodium/read/00-intro.md#installing-libsodium)
+    - open_basedir access to `/dev/urandom`
+    - mcrypt extension (mcrypt needs to be able to access `/dev/urandom`. This means if `open_basedir` is set, it must include this file.)
+    - com_dotnet extension
 - GD extension
-- zlib extension
-- some disk space or (optionally) a database supported by [PDO](https://secure.php.net/manual/book.pdo.php)
+- some disk space or (optionally) a MySQL or SQLite database
```
Comment (mlc) [inline INSTALL.md:21]: elsewhere the docs say Postgres is supported

**Comment by elrido** [comment]
zlib on the server side is always required as it isn't used for decompression, but as an entropy check on every submitted paste (so one can't store unencrypted pastes, making the server operator liable for the content).

https://github.com/PrivateBin/PrivateBin/blob/1ddbcf79ac9845f10969b6a0538d4ae550922a2b/lib/FormatV2.php#L111-L114

The database support really is intended to be generic and the current implementation should cover:
- ibm (DB2)
- informix
- (Microsoft) mssql
- mysql (and of course MariaDB)
- (Oracle) oci
- (Postgres) pgsql
- sqlite

https://github.com/PrivateBin/PrivateBin/blob/1ddbcf79ac9845f10969b6a0538d4ae550922a2b/lib/Data/Database.php#L516-L556

On push to git, we do only test for sqlite in the unit tests and I know of folks using MySQL and PostgreSQL in production. The above linked PDO support source code is mostly taken from Zend Framework 1. I am happy for merge requests that add support for additional PDO drivers and any reports on working or non-working setups using one of drivers other then the known working SQLite, MySQL/MariaDB and PostgreSQL.

**Comment by techboyg5** [comment]
I fixed it!

**Review by rugk — APPROVED** [review]
(Just choose a better branch name the next time :stuck_out_tongue_winking_eye:)

**Pull request merged**